src/components/ErrorLogsView/ErrorLogsView.js

```diff
@@ -35,17 +35,17 @@ export const ErrorLogsViewComponent = ({ resources: { log } }) => {
       data-test-error-logs-container
       className={css.errorLogsContainer}
     >
-      {errorLogRecords.map(errorLogRecord => (
+      {errorLogRecords?.map(errorLogRecord => (
         <div
           key={errorLogRecord.id}
           data-test-error-log
         >
           <div data-test-error-log-info>{errorLogRecord.createdDate} {errorLogRecord.logLevel} {intl.formatMessage(
             { id: `ui-data-export.${errorLogRecord.errorMessageCode}` },
-            { ...formatErrorReasonMessageValues(errorLogRecord.errorMessageValues) }
+            { ...formatErrorReasonMessageValues(errorLogRecord?.errorMessageValues) }
```

Comment:
if it's possible to have undefined/null here, so it's better to have `errorLogRecords?.filter(Boolean).map` as you have other places where you don't use .?, e.g. errorLogRecord.id

Reply:
I just forgot to remove it from there optional changing. I put it at the start of the map, so here it's not necessary.